#

float32[] poly_x
float32[] poly_y
float32[] poly_z
float32[] poly_yaw
duration duration